src/tests/ftest/osa/osa_online_reintegration.yaml

```diff
@@ -45,4 +43,4 @@ ior:
     #   - [scmsize, nvmesize, transfersize, blocksize, PASS/FAIL(Expected) ]
     #    The values are set to be in the multiples of 10.
     #    Values are appx GB.
-    - [4000000000, 18000000000, 4000, 50000000, PASS]          #[4G, 18G, 4K, 51M, PASS]
```

Comment:
Is the removal of the newline at the end of the file intentional?

Reply:
I did not removed it intentionally

Reply:
I am not sure why it got removed but don't think it will matter for execution.